py/desisim/lya_spectra.py

```diff
@@ -104,10 +114,267 @@ def get_spectra(lyafile, nqso=None, wave=None, templateid=None, normfilter='sdss
         normmaggies = np.array(normfilt.get_ab_maggies(padflux, padwave, 
                                                        mask_invalid=True)[normfilter])
         flux1 *= 10**(-0.4 * mag_g[ii]) / normmaggies
+
+        # Inject a DLA?
+        if add_dlas:
+            if np.min(wave/1215.67 - 1) < zqso[ii]: # Any forest?
+                if debug:
```

Comment:
preferable to use `log=desiutil.log.get_logger(); log.debug(message)` rather than `if debug: print...`.  Or remove altogether if you are done debugging.